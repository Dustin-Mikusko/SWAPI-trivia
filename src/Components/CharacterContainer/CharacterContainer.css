.character-container {
  background-image: url('../../images/background-stars.jpeg') ;
  display: flex;
  justify-content: center;
  align-items: center;
  height: 100%;
  flex-wrap: wrap;
  background-attachment: fixed;
}

.character-container {
  
}
