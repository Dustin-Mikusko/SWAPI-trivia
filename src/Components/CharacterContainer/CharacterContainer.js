import React from 'react';
import './CharacterContainer.css';
import Character from '../Character/Character';

const CharacterContainer = (props) => {
  console.log(props.movie);
  const characters = props.movie.characters.map(character => {
    return <Character
      name={character.name}
      world={character.world}
      population={character.world[0].population}
      species={character.species[0]}
      relatedFilms={character.relatedFilms.join(', ')}
    />
  })

  return (
    <div>
      {console.log(characters)}
      {characters}
    </div>
  )
}

export default CharacterContainer;
