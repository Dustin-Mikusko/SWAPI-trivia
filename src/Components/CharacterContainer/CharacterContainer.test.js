import React from 'react';
import { shallow } from 'enzyme';
import CharacterContainer from './CharacterContainer';
