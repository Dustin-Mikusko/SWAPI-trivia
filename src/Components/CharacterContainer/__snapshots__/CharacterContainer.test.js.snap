// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CharacterContainer should render CharacterContainer component with props to match snapshot 1`] = `
<Fragment>
  <Header
    logOut={[MockFunction]}
    user={
      Object {
        "name": "Tom Brady",
        "quote": "My favorite ring is the next one.",
        "rank": "beginner",
      }
    }
  />
  <main
    className="character-container"
  >
    <ScrollingText />
    <header>
      <h1
        className="movie-title"
      >
         Characters
      </h1>
    </header>
    <Character
      key="0"
      name="Tom Brady"
      relatedFilms="Reeses, Pieces, Species"
      species="G"
    />
  </main>
</Fragment>
`;
